eslint/babel-eslint-tests/test/integration/eslint/eslint.js

```diff
@@ -1,3 +1,5 @@
+import eslint from "eslint";
+import * as parser from "../../../../babel-eslint-parser";
```

Comment:
Can we add `@babel/eslint-parser` to the `dependencies` of `@babel/eslint-tests`?

Reply:
Ah, yeah - I thought it wouldn't update when changes were made to the parser, but I just confirmed that's not the case.